app/models/fax.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class Fax
+  delegate \
+    :sfax_api_key,
+    :sfax_encryption_key,
+    :sfax_init_vector,
+    :sfax_username,
+    to: :"Rails.application.secrets"
+
+  def self.send_fax(number:, file:)
+    new(number, file).send_fax
+  end
+
+  def initialize(number, file)
+    @number = number
+    @file = file
+  end
+
+  def send_fax
+    sfax.send_fax(number, file)
+  end
+
+  protected
```

Comment:
Either way sounds good to me, however we should probably be consistent across the codebase. I'm slightly in favor of `private`, but frankly, I'll be happy if they're not fully public `attr_*`s. 

Ping @jessieay for perspective. She's off this week so maybe merge and we can discuss when she is back? Up to you.